### PR TITLE
KOGITO-7245: Patch PR for KOGITO-6998 with doc improvements in Configuring OpenAPI services endpoints guide

### PR DESCRIPTION
**Pull request by hmanwani-rh**
**JIRA:** https://issues.redhat.com/browse/KOGITO-7245

**Description:** Patch PR for KOGITO-6998 with doc improvements in Configuring OpenAPI services endpoints guide

https://github.com/kiegroup/kogito-docs/pull/92

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

### CODE REVIEW

**Comment by hmanwani-rh** [comment]
@hbelmiro Please review the changes made in this PR and share your comments.

**Review by hbelmiro — COMMENTED** [review]
Thanks for your improvements, @hmanwani-rh.
I just found a typo. Everything else LGTM.

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc [hunk 1]
```diff
@@ -418,37 +392,38 @@ curl -X 'POST' \
...
-For more information, see <<proc-running-the-services, Running the services that the Serverless Workflow application will call>>.
+For more information, see <<proc-config-openapi-services-running-the-services, Running the services>>.
 
 .Procedure
-. In a separate terminal window, run the Serverless Workflow application in development mode overwriting the property defined in the `application.properties` file with an environment variable:
-
+. In a separate command terminal window, run the Serverless Workflow application in development mode, overwriting the property defined in the `application.properties` file using an environment variable:
 +
 --
+.Run the Serverless Workfloa application in development mode
```
[the remark this answers is not in the document]
Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc]: Done

**Comment by hmanwani-rh** [comment]
> Thanks for your improvements, @hmanwani-rh. I just found a typo. Everything else LGTM.

@hbelmiro Thanks for reviewing the changes. I've fixed the typo. 
Merging this PR.

**Pull request merged**